src/data/scriptTemplates.ts

```
@@ -0,0 +1,957 @@
+            displayName: {
+              en: 'Presenter'
+            },
+            voiceId: 'shimmer'
+          }
+        }
+      }
+    },
+    scriptName: 'movie_prompts_no_text_template.json',
+    systemPrompt: 'Generate a script for a movie trailer of the given story. Another AI will generate images for each beat based on the image prompt of that beat. Movie prompts must be written in English.',
+    title: 'Movie Trailer template'
+  }
```

ESLint

[error] Insert `,`

(prettier/prettier)